src/com/tutorial/main/Handler.java

```diff
@@ -1,51 +0,0 @@
-package com.tutorial.main;
-
-import java.awt.Graphics;
-import java.util.ArrayList;
-import java.util.LinkedList;
-
-public class Handler {
-	
-	LinkedList<GameObject> object = new LinkedList<GameObject>();
-	LinkedList<GameObject> toAdd = new LinkedList<GameObject>();
```

Comment:
If you look in Player.java, you will see that I iterate over handler.object to check for collisions. I can't think of an efficient way to check for collisions between objects. What I can think of is O(n!) if checking all pair-wise collisions between objects instead of O(n) checking only for player vs something else collisions. So if I use this outside of Handler, would I make it public or protected or just find some way to NOT use it outside of Handler?